src/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { Badge } from './Badge';
+
+export default {
+  title: 'UI-Library/Badge',
+  component: Badge,
+  //argTypes: {},
+} as ComponentMeta<typeof Badge>;
+
+const Template: ComponentStory<typeof Badge> = args => <Badge {...args} />;
+
+export const BadgeMain = Template.bind({});
+BadgeMain.args = {
+  size: 24,
```

Comment:
once you have changed the size prop, you should use argTypes on line #8 to set it as a radio or drop-down, refer to examples in accordion story or button